packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page.tsx

```diff
@@ -82,23 +84,23 @@ const SimpleEntityPage: NextPage = () => {
           />
         )}
       </div>
-    </MainContentWrapper>
+    </>
   );
 };
 
-const BlockBasedEntityPage: NextPage = () => {
+SimpleEntityPage.getLayout = getLayoutWithSidebar;
+
+const BlockBasedEntityPage: NextPageWithLayout = () => {
   const router = useRouter();
   const { query } = router;
   const { accountId } = useRouteAccountInfo();
   const entityId = query["entity-id"] as string;
 
-  return (
-    <MainContentWrapper>
-      <BlockBasedEntityEditor accountId={accountId} entityId={entityId} />
-    </MainContentWrapper>
-  );
+  return <BlockBasedEntityEditor accountId={accountId} entityId={entityId} />;
 };
 
+BlockBasedEntityPage.getLayout = getLayoutWithSidebar;
+
 export default process.env.NEXT_PUBLIC_BLOCK_BASED_ENTITY_EDITOR === "true"
   ? BlockBasedEntityPage
   : SimpleEntityPage;
```

Comment:
[optional]

```
export default process.env.NEXT_PUBLIC_BLOCK_BASED_ENTITY_EDITOR === "true"
  ? BlockBasedPage
  : Page;
```

Reply:
Addressed in [8c19e64](https://github.com/hashintel/hash/pull/468/commits/8c19e64f44905c778cdeca75b7c8d919b5e779b3)